docker-entrypoint.sh

```diff
@@ -1,6 +1,22 @@
 #!/usr/bin/env sh
 set -e
 
+if [ ! -d "/dev/longhorn" ]; then
+	echo "Failed to find longhorn directory"
+	exit 1
+fi
+
+device_dir=$(ls -t /dev/longhorn/ | head -n 1)
```

Comment:
This is no safer than before. If there are multiple volumes created near the same time, this will break and write into other volumes.

I don't have a good idea right now due to the Kubernetes issue. But at least we should ensure there is only one device under `/dev/longhorn` before proceeding.

Also, we can add a warning in the README to make sure there is no other volumes are created using Longhorn when the test is running.

It would be best if we can figure out what's the PVC associated with the pod, but that would require a service account + Kubernetes API client to do so.

Reply:
Let me update `README` for now to make sure there should be any volume before running this test. And I will think about how to improve this later.

Reply:
There is a check after the `/dev/longhorn` directory, to make sure at least we get one block device name.

Reply:
I am worried about getting the wrong device name. So we need to make sure there is only one device in that directory, otherwise, this can write into any running workload, result in corrupting the data.

Reply:
I see, updated the script. Thanks.